src/Left-menu.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import SelectLeftMenu from './select-left-menu.js';
+import ListMovies from './list-movies.js';
+
+class LeftMenu extends Component {
+    render () {
+        return (
+            <div className="left-menu">
+                <SelectLeftMenu />
```

Comment:
This component is not necessary, please remove it and put the html inside LeftMenu component.